app/controllers/api/v0x1/tasks_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  module V0x1
+    class TasksController < Api::V0x0::TasksController
+      include Api::V0x1::Mixins::IndexMixin
+
+      def update
+        model.update(params.require(:id), params_for_update)
+
+        messaging_client.publish_message(
+          :service => "platform.topological-inventory.events",
```

Comment:
Since this is going to be specific to task updates (similar to how the persister emits inventory updates) we can be consistent with the naming of that topic so how about `platform.topological-inventory.task-output-stream`?